libbpf-rs/Cargo.toml

```diff
@@ -15,9 +15,8 @@ keywords = ["bpf", "ebpf", "libbpf"]
 maintenance = { status = "actively-developed" }
 
 [features]
-# When turned on, link against system-installed libbpf instead of building
-# and linking against vendored libbpf sources
-novendor = ["libbpf-sys/novendor"]
+# When turned on, compile and link all libbpf library dependencies statically
+# When turned off, dynamically link all libbpf library dependencies
```

Comment:
Let's please also mention that `static` implies vendoring of all C library dependencies? That is crucial from my perspective.